tests/screenshots/Cargo.toml

```diff
@@ -18,9 +18,11 @@ slint = { path = "../../api/rs/slint", default-features = false, features = ["st
 i-slint-core = { version = "=0.3.4", path = "../../internal/core" }
 i-slint-backend-testing = { path = "../../internal/backends/testing" }
 image = { version = "0.24.0", default-features = false, features = ["png", "jpeg"] }
+base64 = "0.20"
```

Comment:
You're not using base64.

(I was using it to show the result in the console in a more condensed form than a list of u8)